### ISSUE TITLE
Playlist Entries

### ISSUE THREAD

**Issue by Eikthyr** [iBicha/UnityYoutubePlayer#63]
var playList = await YoutubeDl.GetVideoMetaDataAsync<YoutubePlaylistFlatMetadata>(playlistUrl,
      YoutubeDlOptions.FlatPlaylist, k_PlaylistFields);

  PlaylistTitle.text = playList.Title;

  foreach (var entry in playList.Entries) -->> there is an error in this part
......
.....
....

I think there is a problem in connecting the videos in the playlist from the plugin.playList doesn't get any information except the title.

**Comment by iBicha** [comment]
If you are referring to the [draft PR](https://github.com/iBicha/UnityYoutubePlayer/pull/60/files), what is the error?
And it is expected to only get the url and the title

**Comment by Eikthyr** [comment]
that's what i didn't understand.i started the project without making any changes but i am getting error. There is no problem in other scenes, but in the part I specified, the entries return null.i am using 2020.2 as unity version

**Comment by Eikthyr** [comment]
![Error](https://user-images.githubusercontent.com/84289107/129438825-cfe60127-d8d9-4309-80db-ed3a3068d5c2.png)
any chance to fix it?

**Comment by iBicha** [comment]
That feature is a work in progress, but leave the url to the playlist, and I'll try to investigate when I get some time

**Comment by Eikthyr** [comment]
I understand I will be waiting. Thanks for your interest and making this plugin

**Comment by Eikthyr** [comment]
I solved the problem.There is no 'playlist' at my link. Youtube does not see someone else's list as a 'playlist'.
e.g 
--- someone else's list someone ---
https://www.youtube.com/watch?v=a24u_ACKnvs&list=UUJBpeNOjvbn9rRte3w_Kklg
--- my list ---
https://www.youtube.com/playlist?list=PL6gJDG2WN9JoJCAw0lyCPTvssUgLVDgtP

I think the plugin is giving an error because of this 'watch' part.If I change the link from 'watch' to 'playlist', the problem disappears.
https://www.youtube.com/playlist?list=UUJBpeNOjvbn9rRte3w_Kklg

I hope I have helped others.thanks again
